ctpolicy/ctpolicy.go

```diff
@@ -29,20 +29,20 @@ const (
 
 // LogGroupInfo holds information on a single group of logs specified by Policy.
 type LogGroupInfo struct {
-	name          string
+	Name          string
 	LogURLs       map[string]bool // set of members
-	minInclusions int             // Required number of submissions.
-	isBase        bool            // True only for Log-group covering all logs.
+	MinInclusions int             // Required number of submissions.
+	IsBase        bool            // True only for Log-group covering all logs.
 }
 
 func (group *LogGroupInfo) setMinInclusions(i int) error {
 	if i < 0 {
 		return fmt.Errorf("cannot assign negative minimal inclusions number")
 	}
 	// Assign given number even if it's bigger than group size.
-	group.minInclusions = i
+	group.MinInclusions = i
 	if i > len(group.LogURLs) {
-		return fmt.Errorf("trying to assign %d minimal inclusion number while only %d logs are part of group %q", i, len(group.LogURLs), group.name)
+		return fmt.Errorf("trying to assign %d minimal inclusion number while only %d logs are part of group %q", i, len(group.LogURLs), group.Name)
```

Comment:
I'd love to have this as warning, yet do the assignment. No idea how to express warnings here properly.

Reply:
I think setters should prevent the object from getting into an invalid state so errors are detected early. IMO it should not accept this invalid value as it can break things later.

Reply:
That's not an invalid state. That's possible state. The fact that we won't be able to achieve enough SCTs within logGroup provided is important (that's why error/warning), but it shouldn't prevent us from building such policy upon request.

Reply:
OK but any time you do this it's something extra that people using the code have to be aware of.